docs/reference/index-modules/allocation/delayed.asciidoc

```diff
@@ -38,14 +38,15 @@ This setting can be updated on a live index (or on all indices):
 
 [source,js]
 ------------------------------
-PUT /_all/_settings
+PUT _all/_settings
 {
   "settings": {
     "index.unassigned.node_left.delayed_timeout": "5m"
   }
 }
 ------------------------------
 // AUTOSENSE
+// TEST[s/^/PUT test\n/]
```

Comment:
The snippet fails without there being some index present who's setting we can modify. It doesn't feel important to the docs that we mention that so I just slapped a command to create the index at the front of the test.